Command/InvalidateTagCommand.php

```diff
@@ -32,13 +38,14 @@ class InvalidateTagCommand extends BaseInvalidateCommand
      * If no cache manager is specified explicitly, fos_http_cache.cache_manager
      * is automatically loaded.
      *
-     * @param CacheManager|null $cacheManager The cache manager to talk to.
-     * @param string            $commandName  Name of this command, in case you want to reuse it.
+     * @param TagHandler|null $tagHandler  The tag handlerto talk to.
+     * @param string          $commandName Name of this command, in case you want to reuse it.
      */
-    public function __construct(CacheManager $cacheManager = null, $commandName = 'fos:httpcache:invalidate:tag')
+    public function __construct(TagHandler $tagHandler = null, $commandName = 'fos:httpcache:invalidate:tag')
```

Comment:
The BC way would be to allow both CacheManager and TagHandler as constructor arguments, check the type and throw an exception if it is neither of both.